CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 All notable changes to this project will be documented in this file.
 
 ## Unreleased
+* `opentelemetry-exporter-collector-proto`
```

Comment:
changelog is generated automatically, you shouldn't need to add it

Reply:
Sweet thanks, I didn't know that!